b2/console_tool.py

```diff
@@ -590,7 +590,7 @@ class Sync(Command):
     """
     b2 sync [--delete] [--keepDays N] [--skipNewer] [--replaceNewer] \\
             [--compareVersions <option>] [--threads N] [--noProgress] \\
-            [--excludeRegex <regex>] <source> <destination>
+            [--excludeRegex <regex>] [--includeRegex <regex>] <source> <destination>
```

Comment:
Applied requested changes; added note to Sync class documentation about --includeRegex requiring --excludeRegex for clarity.